ADOPTERS.md

```diff
@@ -11,6 +11,7 @@ URLs if possible) and [submit a pull request][1]!
 * [Chartboost](https://www.chartboost.com/)
 * [Crimson Hexagon](http://www.crimsonhexagon.com/)
 * [DICE](http://dice.se/)
+* [Despegar](http://www.despegar.com)
```

Comment:
also, could you put this line about the DICE line so we can stay in alphabetic order?